### DISCUSSION TITLE
Final project

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#81] [Assignments]
- Group name
- Group members
- Final project title
- Final project short description
- Final project link
- Final project presentation
- Additional materials

**Comment by kimjee8955** [comment]
- Datart
- Annmarie Cong, Eustina Kim, Michelle Lee
- Datart is a digital mapping website depicting the progression of Los Angeles public art throughout time. Our dataset included over 1,500 art pieces dated from 1925-2017. To visualize our data, we developed various maps, charts, and image galleries. We also implemented other cool features to make it more interactive and exciting to users.
- https://kimjee8955.github.io/Datart/
- https://docs.google.com/presentation/d/1N5JeIMyqX-Y78zTXRjocvzJKJLpu5dnQ8uFrkdFqvUU/edit?usp=sharing

**Comment by MaryJaneHowland** [comment]
- Group: Happy Tweet
- Members: Ivan Ma, Josephine Meyer-Hogan, MJ Howland
- Title: Happy Tweet
- Description: Happy Tweet is a website focused on digging deeper into the connection between social media and happiness levels. We are interested in seeing how emotions are expressed in social media, specifically via Twitter for the purpose of seeing if there is any correlation between happiness scores and Twitter content.
- Link to website: https://ivanma9.github.io/HappyTweet/
- Link to presentation: https://docs.google.com/presentation/d/1GlYSQ-yxZvzY-zJ1iO6JjAlE0tEquWPCZ84ysuQH5dA/edit?usp=sharing
- Link to final proposal: https://github.com/ivanma9/HappyTweet/blob/main/Group%20Assignments/FinalProjectProposal.md

**Comment by lisalou36** [comment]
- Group name: Environmental Racism in Carceral California
- Group members: Hannah Lien, Julia Tanenbaum, Lisa Kahn
- Final project title: Environmental Racism in Carceral California
- Final project short description: This project's website shows the severe presence of multiple environmental hazards throughout California, the location and overwhelming number of prison facilities throughout California, and the correlation between prison locations and environmentally toxic regions. We hope that visualizing this correlation will encourage site visitors to think about how prison abolition, climate justice, and environmental racism are interlinked.
- Final project link: https://jmtanenbaum.github.io/DHEnviroIncarceration/index.html
- Final project presentation: https://docs.google.com/presentation/d/1cdXKsVTubNpTduznyoZ_4jy7YQgiwdkm3kIZvvFpCP8/edit?usp=sharing

**Comment by JuliaWood1** [comment]
- Group name: Ocean Pollution Posse
- Group members: Derek Wang, Eli Aranita, Julia Wood
- Final project title: How GDP Affects Mismanaged Plastic Waste
- Final project short description: We wanted to look at mismanaged plastic waste worldwide, initially we were interested to develop our narrative around  fishing and plastic pollution. However, we transitioned to analyzing how a country's Gross Domestic Product (GDP) affects their mismanaged plastic  waste.
- Final project link: https://eliaranita1.github.io/project1/landing.html
- Final project presentation: https://docs.google.com/presentation/d/1KTLVA4ByiMWpaApIkvuRMF43R9kHoaiLUBNr7oBuCMc/edit?usp=sharing

**Comment by alicelu170** [comment]
Group name: Rethink Recycle
Group members: Alice Lu, Hozumi Oshida, Emily Chau
Final project title: Rethink Recycle
Final project short description: The goal of Rethink Recycle is to help users understand the scope of the global waste-management network. The true impact of plastic waste goes beyond sorting one’s recycling; our goal is that Rethink Recycle will make this reality clearer and will provide visitors with resources.
Final project link: https://hoz-map.github.io/Rethink-Recycling/Final/index.html
Final project presentation: https://docs.google.com/presentation/d/1DOM3XKQOHfRtOymrQ2DXA2nAqK4sXcdHTifmsDq0sRM/edit?usp=sharing

**Comment by ryanorlando** [comment]
- Group Name: Team Data
- Group Members: Ryan Orlando, Ainsleigh Douglas, Sebastien Dills
- Final Project Title: The Weight of Data
- Description: Our final project looks at the usage of the internet, data, and data centers and how this technology impacts the environment around the world. We added several features aimed to engage the user and educate them on the connection between data and carbon emissions. We hope that this site will inspire users to think more about their impact on the environment as they use the internet.
- Final Project Link: https://ainsleighdouglas.github.io/DH-151/Final/landing.html
- Final Project Presentation: https://docs.google.com/presentation/d/1MgcGNnUJ47qQ94GmZa1M9Wp6hgm020YfX5VppbfEhMY/edit#slide=id.g63da1a4385_0_0

**Comment by alekhyavittalam** [comment]
Group Name: State-of-the-Climate
Group Members: Natasha Grace, Zona Liao, Alekhya Vittalam
Final Project Title: State-of-the-Climate
Description: Using geospatial data sets of different countries, our group focuses on mortality rates as a result of varying levels of air pollution. Household air pollution was responsible for 3.8 million deaths around the world (World Health Organisation, 2016).  We feel that creating a visual location guide could help us gain a clearer understanding as to where air quality could be detrimental for most and the socioeconomic reasons behind it. Income levels are directly related to the access the country has to making environmental decisions (prioritising profits over less pollution) and then subsequently, how sheltered the people are to being exposed to these potential risks (healthcare, discrimination).
Final Project Link: https://alekhyavittalam.github.io/state-of-the-climate/FinalProject/index.html
Final Project Presentation: https://docs.google.com/presentation/d/1vJI6KaQS6iuJIr4hdw6RsmiESZR35APG3He8JVSB6ok/edit?usp=sharing

**Comment by frida-barragan** [comment]
- Health Mappers
- Frida Barragan & Kate McInerny
- Mental Health in Los Angeles County
- We used geographical data to explore the current mental health crisis in Los Angeles County. We mapped mental health facilities and intersected this data with demographic and income information. We also included interactive features such as a landing page, buttons and timelines to capture the user's interest.
- link: https://frida-barragan.github.io/DH151/final/landingpage/index.html
- presentation: https://docs.google.com/presentation/d/1xnqlLzOxQjcZvmLTWbd6GFSU9sg5aLXFYyXpovh7Rp4/edit?usp=sharing

**Comment by kaicolorado** [comment]
- cARTographers
- Alex Haddad & Kai Colorado
- cARTographers: Mapping U.S. Art Education
- Our project explores arts education within the American education and legislative system. We mapped arts education policies and standardized test scores across the United States to visualize the patterns and differences in arts education and academic performance across different states.
- https://kaicolorado.github.io/DH151-Group/
- https://docs.google.com/presentation/d/1b6Eqx0KU_Fk9vo7tS-cTa3d5hFpzTms8_HCHiXa-n4Q/edit#slide=id.gdeec663fa4_1_187

**Comment by adrianaromero819** [comment]
Fantastic Four
Labiba Alam, Haiqi, Adriana Romero, Lynn Tieu
Mapping Heritage: History and Hate
Our map showcases important events in Asian American history and anti-Asian hate incidents that have occurred over the past year. We hope our website encourages people to think holistically about Asian American hate in the U.S. and remain aware.
Presentation: https://docs.google.com/presentation/d/10Nuee1YaoXfA8vkx6ApehIzqA01WqUKmP-a9GIpFDDg/edit?usp=sharing
Website: https://lynnt2.github.io/FantasticFour/Midterm/index.html

**Comment by justisg** [comment]
The Musical Maps
Justis Guardia, Daniel Liu, Kai Watanabe
The Musical Maps: Music During the Pandemic
Our project showcases an interactive map illustrating the listening trends and shifts between the years 2019 and 2020. This shows how the COVID-19 pandemic affected the ways in which users listened to music. This is shown through our data points that represent the concerts that occurred internationally during this time.
Website: https://justisg.github.io/TheMusicalMaps/Website/index.html
Presentation: https://docs.google.com/presentation/d/1SvJS64kwBieYeTzDD-1OTlTT0BcoMr9YNXD17KQwKBc/edit?usp=sharing
